src/dao/SituationFeedbackDAO.ts

```diff
@@ -130,10 +128,10 @@ export class SituationFeedbackDAO extends BaseDAO {
    * @hidden
    */
   private async post(url: string, data: any): Promise<void> {
-    const options = new OnmsHTTPOptions();
-    options.headers['content-type'] = 'application/json';
-    options.headers.accept = 'application/json';
-    options.data = data;
+    const options = new OnmsHTTPOptions()
```

Comment:
Good example here where builder pattern would be nicer since here we created 4 instances of `OnmsHTTPOptions` using the `with` methods.